src/templates/Navigator/index.js

```diff
@@ -1,12 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
 import Icon from 'atoms/Icon';
 import TextComponent from 'atoms/TextComponent';
 import StepProgress from 'molecules/StepProgress';
 import ContactCard from 'organisms/cards/ContactCard';
 import styles from './navigator.module.scss';
 
+
+function renderStepProgress() {
+  return (
+    <StepProgress
+      className={styles['step-progress']}
+      data={[
+        { active: false,
+          inactive: false,
+          icon: 'icon-calculator',
+          label: 'Aenean',
+          link: '#' },
+        { active: true,
+          inactive: false,
+          icon: 'icon-health',
+          label: 'Phasellus',
+          link: '#' },
+        { active: false,
+          inactive: true,
+          icon: 'icon-application',
+          label: 'Curabitur',
+          link: '#' },
+        { active: false,
+          inactive: true,
+          icon: 'icon-quotes',
+          label: 'Etiam',
+          link: '#' }
+      ]}
+    />
+  );
+}
+
+function renderContactCard() {
```

Comment:
Looks like the right rail will need tweaking as well because depending on where you're at in the flow, you might have the map or the card with the policy details in addition to the contact card. Could we just expose the right rail as its own component inside the Navigator so that it's purely presentational?